photoprism/docker-compose.yml

```diff
@@ -5,6 +5,9 @@ services:
     environment:
       APP_HOST: $APP_PHOTOPRISM_IP
       APP_PORT: $APP_PHOTOPRISM_PORT
+      # These endpoints are protected by HTTP Basic Auth by Photoprism.
+      # More details here: https://docs.photoprism.app/user-guide/sync/webdav/#server-url
+      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
   web:
     image: photoprism/photoprism:20210925@sha256:238e6955804d82097d08f0e4318835721f46bd844f9065a38aa7d112ab72906e
```

Comment:
This version is almost a year old? You might want to update it from time to time 🤗

👉 https://docs.photoprism.app/release-notes/